VMDP doc set (#111284)
openshift/openshift-docs: backup_and_restore/application_backup_and_restore/oadp-vmdp/oadp-vmdp.adoc

:_mod-docs-content-type: ASSEMBLY
include::_attributes/common-attributes.adoc[]

[id="oadp-vmdp"]

> = {oadp-short} VM data protection

Reviewer: lets provide full context in title 
```suggestion
= {oadp-short} Virtual Machine data protection
```

> = {oadp-short} Virtual Machine data protection

Reviewer 2: I believe it should be "virtual machine data protection" as opposed to "Virtual Machine data protection"?
Also, almost all headings use acronym (VMDP). I'd use either acronym OR full spelling for all headings.

Author: done.

> :context: oadp-vmdp

Reviewer 2: ```suggestion

:context: oadp-vmdp
```

An empty line after heading is required for AEM and DITA.

toc::[]

[role="_abstract"]

> Use {oadp-short} VM data protection (VMDP) to back up and restore user data from within virtual machines (VMs) on {VirtProductName}. This helps you to protect files and directories independently without relying on cluster administrators.

Reviewer 2: ```suggestion
Use {oadp-full} virtual machine data protection (VMDP) to back up and restore user data from within VMs on {VirtProductName}. This helps you to protect files and directories without relying on cluster administrators.
```

We probably want to expand all acronyms on the first instance?
'Independently' already implies 'without relying on'. How about skipping it? The proposed wording with 252 characters would fit the limit.
